docs/content/articles/es/madara-beast-article.md

```diff
@@ -0,0 +1,166 @@
+![thee BEEAAST](https://imgur.com/EBwBNnB.jpg)
+
+# Aprovechando a la Bestia - Madara y la Revolución de las Appchains de Starknet
+
+**20 de julio de 2023** · 1 min de lectura
+
+<font size=5>_Desde Reducciones Masivas de Costos hasta Control Personalizado,
+Descubre el Futuro de la Infraestructura Blockchain_</font>
+
+---
+
+## TL;DR
+
+- Madara es un secuenciador de alto rendimiento de Starknet, que proporciona el
+  poder para crear
+  [Appchains](https://www.starknet.io/en/posts/ecosystem/the-starknet-stacks-growth-spurt)
+  personalizadas y eficientes.
+- Al utilizar el framework Substrate, Madara amplía las capacidades de la
+  Máquina Virtual Cairo, lo que conduce a programas demostrables, seguros y
+  flexibles.
+- Su implementación ofrece numerosos beneficios como infraestructura escalable,
+  alto rendimiento y un control sin precedentes sobre las aplicaciones.
+- Las características únicas de Madara incluyen soporte para la privacidad
+  potencial en cadena, interoperabilidad simplificada entre diversas cadenas y
+  una ejecución robusta.
+- Madara está abriendo el camino en el desarrollo de dApps al ofrecer soluciones
+  rentables, escalables y personalizables en el dominio de la cadena de bloques.
```

Comment:
La palabra `blockchain` es un anglicismo no? Pero si prefiere usar `cadena de bloques`, también deberíamos de cambiar el título

Reply:
Tenés razón, cambié esto dejándolo en itálica, además aproveché para cambiar "en cadena" por "on-chain" igual en itálica, no me parece que se entienda de otra manera. ¿Qué te parece?